docs/src/main/paradox/scala/http/client-side/index.md

```diff
@@ -17,7 +17,7 @@ Depending on your application's specific needs you can choose from three differe
   the user can supply a ``Source[HttpRequest, NotUsed]`` with requests to run against a single host
```

Comment:
Drop the double back-ticks and add a `@java[Source<HttpRequest, NotUsed>]` (not sure whether Java uses `NotUsed` or `Void`).